src/main/java/org/folio/helper/PurchaseOrderHelper.java

```diff
@@ -292,6 +279,16 @@ public Future<Void> updateOrder(CompositePurchaseOrder compPO, boolean deleteHol
       });
   }
 
+  private Future<List<Void>> validateUserUnaffiliatedPoLineLocations(CompositePurchaseOrder purchaseOrder, RequestContext requestContext) {
+    var poLineFutures = new ArrayList<Future<Void>>();
+    if (!purchaseOrder.getCompositePoLines().isEmpty()) {
```

Comment:
I think check for empty is not need 
1) we are invoking after empty check line if (isTransitionToOpen && CollectionUtils.isEmpty(compPO.getCompositePoLines())) { and 
2) we are using filter

However, if it is better to use incase of invoking in other places in future lets use CollectionUtils.isNotEmpty(compPO.getCompositePoLines()  to be easier to read

Reply:
Better to leave this slightly excessive null checks because the method is also used for Unopen and also can be called from the Postman where there may not be any PoLines in the request to validate